src/classes/Countertop.js

```diff
@@ -40,13 +48,69 @@ class Countertop {
 			},
 		)
 		this.stations.push(station)
+		this.updateTopology()
 		return station
 	}
 
+	/**
+	 * Start the countertop and begin data processing.
+	 *
+	 * This will regenerate the countertop topology to reflect all currently added appliances.
+	 *
+	 * @return {Boolean} Whether the countertop successfully started.
+	 */
+	start = async () => {
+		this.logger.trace('CountertopCoordinator: start()')
+		this.state = countertopStates.STARTING
+		const promises = this.stations.map(async (station) => station.start())
+		if ((await Promise.all(promises)).every((result) => result)) {
```

Comment:
Okay, so `station.start()` returns a promise with a boolean indicating success or failure? Failure isn't indicated by a promise rejection? I could go look, I guess, but I'm just thinking this one through out loud instead.